q2_diversity/plugin_setup.py

```diff
@@ -146,6 +147,8 @@
     },
     parameter_descriptions={
         'metric': 'The beta diversity metric to be computed.',
+        'pseudocount': ('A pseudocount to handle zeros '
```

Comment:
Maybe we should indicate that this value is ignored in other cases:

> A pseudocount to handle zeros when computing compositional metrics, otherwise, this parameter is ignored.